content/docs/3.0/faq/faq.md

```diff
@@ -0,0 +1,151 @@
+
+PD([Placement Driver](https://github.com/pingcap/pd))  periodically checks replication constraints and balances the load, and it handles data movement automatically. When PD notices that the load is too high, it will rebalance data.
+
+## How is TiKV highly available?
+
+TiKV is self-healing. With its strong consistency guarantee, whether it’s data machine failures or even downtime of an entire data center, your data can be recovered automatically.
+
+## How is TiKV strongly-consistent?
+
+Strong consistency means all replicas return the same value when queried for the attribute of an object. TiKV uses the [Raft consensus algorithm](https://raft.github.io/) to ensure consistency among multiple replicas. TiKV allows a collection of machines to work as a coherent group that can survive the failures of some of its members.
+
+## Does TiKV support distributed transactions?
+
+Yes. The transaction model in TiKV is inspired by Google’s Percolator, a paper published in 2006. It’s mainly a two-phase commit protocol with some practical optimizations. This model relies on a timestamp allocator to assign monotone increasing timestamp for each transaction, so the conflicts can be detected.  
+
+## Does TiKV have ACID semantics?
+
+Yes. ACID semantics are guaranteed in TiKV:
+
+* Atomicity: Each transaction in TiKV is "all or nothing": if one part of the transaction fails, then the entire transaction fails, and the database state is left unchanged. TiKV guarantees atomicity in each and every situation, including power failures, errors, and crashes. 
+* Consistency: TiKV ensures that any transaction brings the database from one valid state to another. Any data written to the TiKV database must be valid according to all defined rules, including constraints, cascades, triggers, and any combination thereof. 
+* Isolation: TiKV provides snapshot isolation (SI), snapshot isolation with lock (SQL statement: SELECT ... FOR UPDATE), and externally consistent reads and writes in distributed transactions. 
+* Durability: TiKV allows a collection of machines to work as a coherent group that can survive the failures of some of its members. So in TiKV, once a transaction has been committed, it will remain so, even in the event of power loss, crashes, or errors. 
+
+## How are transactions in TiKV lock-free?
+
+TiKV has an optimistic transaction model, which means the client will buffer all the writes within a transaction, and when the client calls commit function, the writes will be packed and sent to the servers. If there is no conflict, the writes which are the key-value pairs with specific version are written into the database, and can be read by other transactions.
+
+## Can I use TiKV as a key-value store?
+
+Yes. For now, we have a [TiKV driver for Go](https://github.com/pingcap/tidb/tree/master/store/tikv). You are welcome to develop drivers in other languages!
+
+## How does TiKV compare to traditional relational databases like Oracle or MySQL?
```

Comment:
No, you should not compare tikv to oracle or mysql.

Reply:
What do you suggest? Remove this question?

Reply:
I suggest to answer it with “TiKV is a distributed transactional key-value database, while Oracle and MySQL are relational databases. TiDB may be more suitable to be compared with them."

Reply:
> TiDB may be more suitable to be compared with them.
In this case I don't think this question is worth listing here... So I suggest removing it.